### PR TITLE
docs: add placeholder query to playground

### PR DESCRIPTION
**Pull request by moksha256**
To address #757.

Tested locally and worked fine.

Placeholder query is currently `btcPriceList` since it doesn't require authentication. This query also uses a variable, which may be useful for people to see. But let me know if you would prefer a different query.

### CODE REVIEW

**Comment by nicolasburtey** [comment]
tests are not passing @moksha256 . you'll need to run prettier. `yarn prettier-write` should do it.

**Comment by moksha256** [comment]
> `yarn prettier-write` should do it.

Thanks, all checks now passed.

**Comment by samerbuna** [comment]
Let's put the query text in a variable using a template string (using backticks and dropping the `\n`).  Define it in `src/graphql/playground.ts` and import it in `graphql-server.ts`

**Review by samerbuna — CHANGES_REQUESTED** [review]
Let's put the query text in a variable using a template string (using backticks and dropping the `\n`).  Define it in `src/graphql/playground.ts` and import it in `graphql-server.ts`

**Review by vindard — DISMISSED** [review]
Looks good. Only suggestion I'd add is to potentially also have a `name` property on the tab, maybe something like `btcPriceList Sample Query`. Not a blocker though.

**Comment by moksha256** [comment]
@samerbuna @vindard thanks for the feedback. I added a tab name and defined all 3 query-related strings in `src/graphql/playground.ts` since it seemed like they should be together.

File: src/graphql/playground.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+        formattedAmount
+      }
+    }
+}`
+
+const defaultQueryName = "btcPriceList (Sample Query)"
+
+const defaultQueryVars = '{"range": "ONE_DAY"}'
+
+export { defaultQuery, defaultQueryName, defaultQueryVars }
```
Comment (vindard) [inline src/graphql/playground.ts:17]:
```suggestion
export const BTC_PRICE_QUERY = `query btcPriceList($range: PriceGraphRange!) {
  btcPriceList(range: $range) {
      timestamp
      price {
      base
      offset
      currencyUnit
      formattedAmount
    }
  }
}`

export const playgroundTabs = {
  default: {
    query: BTC_PRICE_QUERY,
    name: "btcPriceList (Sample Query)",
    variables: JSON.stringify({ range: "ONE_DAY" }),
  },
}

```

File: src/servers/graphql-server.ts [hunk 2]
```diff
@@ -147,7 +148,17 @@ export const startApolloServer = async ({
...
     playground: apolloConfig.playground
-      ? { settings: { "schema.polling.enable": false } }
+      ? {
+          settings: { "schema.polling.enable": false },
+          tabs: [
+            {
+              endpoint: "https://api.staging.galoy.io/graphql",
+              query: defaultQuery,
+              name: defaultQueryName,
+              variables: defaultQueryVars,
```
Comment (vindard) [inline src/servers/graphql-server.ts:158]:
```suggestion
              ...playgroundTabs.default,
```

File: src/servers/graphql-server.ts [hunk 3]
```diff
@@ -31,6 +31,7 @@ import {
   ENDUSER_ALIAS,
 } from "@services/tracing"
 import { AccountsRepository } from "@services/mongoose"
+import { defaultQuery, defaultQueryName, defaultQueryVars } from "../graphql/playground"
```
Comment (vindard) [inline src/servers/graphql-server.ts:34]:
```suggestion
import { playgroundTabs } from "../graphql/playground"
```

**Review by vindard — COMMENTED** [review]
Oh that makes sense collecting all the values in one file. I'm suggesting an alternative layout here that we can optionally use as well.

**Comment by moksha256** [comment]
Not sure if anyone else has feedback for vindard's changes so I just committed them and tested...looks good from my end.

**Review by vindard — APPROVED** [review]

**Review by nicolasburtey — APPROVED** [review]

**Comment by vindard** [comment]
> Let's put the query text in a variable using a template string (using backticks and dropping the `\n`).

@samerbuna how would you drop the `\n` from the query string here?

**Pull request merged**